products/grafana.md

```diff
@@ -48,8 +48,8 @@ releases:
     support: 2022-06-13
     eol: false
     releaseDate: 2021-06-08
-    latest: "8.5.15"
-    latestReleaseDate: 2022-10-25
+    latest: "8.5.20"
```

Comment:
LGTM : tags 8.5.16 to 8.5.19 does not exist (https://github.com/grafana/grafana/releases/tag/v8.5.19).